### PR TITLE
*: Use non-root

### PR DESCRIPTION
**Pull request by brancz**
@ant31

### CODE REVIEW

File: contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/templates/grafana-deployment-template.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ spec:
       labels:
         app: grafana
     spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
```
Comment (ant31) [inline contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/templates/grafana-deployment-template.yaml]: what's that id?

Comment (brancz) [reply contrib/kube-prometheus/hack/grafana-dashboards-configmap-generator/templates/grafana-deployment-template.yaml]: > The UID 65534 is commonly reserved for nobody, a user with no system privileges, as opposed to an ordinary (i.e., non-privileged) user. This UID is often used for individuals accessing the system remotely via FTP (file transfer protocol) or HTTP (hypertext transfer protocol).

**Comment by brancz** [comment]
Do not merge this yet, I discovered there are some filesystem issues with the grafana pods.

**Comment by brancz** [comment]
@ant31 this is now ready for review

**Pull request merged**